knowledge-base/kb-buttons-distorted-or-pixelated-buttons.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Fix Pixelated or Distorted Buttons
+page_title: How to Fix Pixelated Buttons
```

Comment:
You can include the word DELL in one of the three tags - title, page_title or description.

You need to include as much information as possible in these tags. By that I don't mean to fill them with all kinds of stuff, but to try make it both short and also detailed by including all key words of the problem. This is needed because the content of the tags is used by the Google SEO which means that it is going to be easier to find the KBs by searching in Google.
Also, it is recommended to use different words in the title, page_title and description when possible. For example:

title - Fix Pixelated or Distorted Buttons on DELL Machines
page_title - Resolve Stretched Pixels in Telerik RadButton
description - "Avoid pixelated or distorted pixels rendering when using Telerik WPF controls on DELL Machines."

I would also include the Telerik word in the title or page_title.